client/my-sites/site-settings/settings-newsletter/ReplyToSetting.tsx

```diff
@@ -0,0 +1,61 @@
+import { FormLabel } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import FormFieldset from 'calypso/components/forms/form-fieldset';
+import FormRadio from 'calypso/components/forms/form-radio';
+
+type ReplyToSettingProps = {
+	value?: string;
+	disabled?: boolean;
+	updateFields?: ( fields: { [ key: string ]: unknown } ) => void;
+	isWPcomSite?: boolean;
```

Comment:
Maybe let's remove `isWPcomSite` prop and use:
```js
const isWPcomSite = useSelector( ( state ) => {
		return ! isJetpackSiteSelector( state, siteId );
} );
```
directly in this component?

Reply:
Good idea!

Reply:
Thanks!